.ado/continuous.yml

```diff
@@ -2,10 +2,11 @@ name: RNW CI $(Date:yyyyMMdd).$(Rev:r)
 
 schedules:
 - cron: "0 11 * * *" # 11AM Daily UTC (3AM Daily PST)
-  displayName: Nightly master publish build
+  displayName: Nightly CI verification build
   branches:
     include:
       - master
+      - 0.65-stable
```

Comment:
Is it possible to manage this within ADO and not in the file here? So we don't have to keep adding branches to the file?